CHANGES.md

```diff
@@ -1,16 +1,24 @@
-# next -- TBA
+# Version 1.2 -- 2024-08-30
 
 ## New Features
 
 * Add `mir_str_slice_value` and `mir_str_slice_range_value` functions, which
   allow taking `&str` slices. For more information, see the documentation in the
   [SAW manual](https://github.com/GaloisInc/saw-script/blob/master/doc/manual/manual.md#string-slices).
 
+* Add `mir_mux_values` command for muxing two MIR values.
+
+* Enhancements to type error messages.
```

Comment:
This overlaps enough with the "The locations printed with type errors (in particular) and other diagnostics [...]" bullet point below that I wonder if we'd be better served by combining them. Perhaps something like:

> * Error messages have been improved such that the locations printed with type errors and other diagnostics now have a much stronger connection with reality.